### PR TITLE
Some changes in hi.js - by BluZed#9946

### PR DESCRIPTION
**Pull request by BluZed**
Just some lines which i discovered while scrolling through

### CODE REVIEW

**Review by DEVILisOP — COMMENTED** [review]
वोल बैंग्स
रैंक  मे  स्पैक्टेट नहीं कर  सक्ते
दूर का शॉट
ign - DEVILisOP

**Review by DEVILisOP — COMMENTED** [review]
संदिग्ध की अमान्य जानकारी
ign - DEVILisOP

**Comment by BluZed** [comment]
" वोल बैंग्स " can be considered
" रैंक मे स्पैक्टेट नहीं कर सक्ते " this does not feel right to me, the other reviewers will have a say on this
" दूर का शॉट " this too does not feel right, "शॉट" is an english term and i think it would be the best if we try to avoid that as we are translating.
" संदिग्ध की अमान्य जानकारी " this is great, editing the pr.

Thanks for the review @DEVILisOP 
We can wait for more reviews to come i guess.

File: hi.js [hunk 1]
```diff
@@ -925,14 +925,14 @@ ranked.join Joined Ranked Match. Abandoning a match will result in a temporary b
 ranked.progress खेल चल रहा है
 ranked.team कृपया एक टीम जॉइन करें
 ranked.switch खेल के बीच मे नहीं बदल सकते
-ranked.spectate Spectating Disabled in Ranked
```
Comment (TheDuce) [inline hi.js:928]: why was Spectating removed here?

File: hi.js [hunk 2]
```diff
@@ -902,15 +902,15 @@ settings.quality.flapAnim स्नाइपर फ्लैप एनीमे
 chall.name.hs Get {0} हैडशॉट्स {1}
 chall.name.ns Get {0} नो-स्कोप्स {1}
 chall.name.ls Get {0} लेगशॉट्स {1}
-chall.name.tmk Get {0} throwing knife kills {1}
-chall.name.fb Get {0} first bloods {1}
+chall.name.tmk Get {0} चाक़ू फेंकने से किल्स {1}
+chall.name.fb Get {0} पहला खून {1}
```
Comment (TheDuce) [inline hi.js:906]: पहला खून wording doesn't go well, maybe make it पहला किल

File: hi.js [hunk 3]
```diff
@@ -902,15 +902,15 @@ settings.quality.flapAnim स्नाइपर फ्लैप एनीमे
...
 chall.name.fk Get {0} हाँथों द्वारा किये गए किल {1}
 chall.name.qs Get {0} क्विक-स्कोप्स {1}
-chall.name.wb Get {0} wallbangs {1}
-chall.name.airk Get {0} mid-air kills {1}
-chall.name.lns Get {0} longshots {1}
-chall.name.tk Get {0} turbo kills {1}
+chall.name.wb Get {0} वालबैंग्स {1}
+chall.name.airk Get {0} हवा में किल्स {1}
+chall.name.lns Get {0} दूर के किल्स {1}
+chall.name.tk Get {0} तेज़ किल्स {1}
```
Comment (TheDuce) [inline hi.js:913]: टर्बो किल would be a better alternative

File: hi.js (hunk 1 above)
Comment (BluZed) [reply hi.js:928]: dont know, gotta add back

File: hi.js (hunk 2 above)
Comment (BluZed) [reply hi.js:906]: okay, changing that

File: hi.js (hunk 3 above)
Comment (BluZed) [reply hi.js:913]: maybe

Comment (DEVILisOP) [reply hi.js]: ye turbo kill makes sense tej kill kinda wierd

**Review by TheDuce — APPROVED** [review]
seems perfect

**Comment by BluZed** [comment]
Ready to merge Matt! 
IGNs ->

@DEVILisOP  -> `DEVILisOP`
@TheDuce -> `vpngaming_69`
@BluZed -> `BluZed`

**Review by ThomasShelby69 — APPROVED** [review]
Good to merge.

File: hi.js [hunk 4]
```diff
@@ -902,15 +902,15 @@ settings.quality.flapAnim स्नाइपर फ्लैप एनीमे
 chall.name.hs Get {0} हैडशॉट्स {1}
 chall.name.ns Get {0} नो-स्कोप्स {1}
 chall.name.ls Get {0} लेगशॉट्स {1}
-chall.name.tmk Get {0} throwing knife kills {1}
-chall.name.fb Get {0} first bloods {1}
+chall.name.tmk Get {0} चाक़ू फेंकने से किल्स {1}
+chall.name.fb Get {0} पहला किल {1}
```
Comment (TheDuce) [inline hi.js:906]: make it pehle kills

**Comment by mty22** [comment]
see https://github.com/mty22/krunker.io-translations/pull/458

**Pull request closed without merging**